collect_app/src/main/res/layout/rating_widget_answer.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:paddingStart="@dimen/margin_small"
```

Comment:
All these padding values can just be replaced with one `padding` as they are all the same!